webview/src/plots/components/comparisonTable/ComparisonTableRows.tsx

```diff
@@ -37,31 +37,33 @@ export const ComparisonTableRows: React.FC<ComparisonTableRowsProps> = ({
     dispatch(changeRowHeight(firstRowHeight))
   }
 
-  const rows = rowsOrder.map((path, i) => {
-    const plot = plots.find(p => p.path === path)
-    if (!plot) {
-      return
-    }
-    const revs = plot.revisions
-    return (
-      <tbody
-        data-testid="comparison-table-body"
-        key={path}
-        id={path}
-        ref={i === 0 ? firstRowRef : undefined}
-      >
-        <ComparisonTableRow
-          path={path}
-          plots={columns.map(column => ({
-            id: column.id,
-            imgs: revs[column.id]?.imgs
-          }))}
-          nbColumns={columns.length}
-          pinnedColumn={pinnedColumn}
-        />
-      </tbody>
-    )
-  })
+  const rows = rowsOrder
```

Comment:
[F] `rowsOrder` can contain `path`s that no longer exist because it is updated by `setState`.